### PR TITLE
added property check parameter to service validate branch

### PR DESCRIPTION
**Pull request by DialgicMew**

### CODE REVIEW

File: plato-api/src/main/java/org/techpleiad/plato/api/response/ServicesAcrossBranchValidateResponseTO.java [hunk 1]
```diff
@@ -12,4 +12,5 @@
 public class ServicesAcrossBranchValidateResponseTO {
     private String service;
     private List<BranchProfileReportResponseTO> report;
+
```
Comment (ashishgopalhattimare) [inline plato-api/src/main/java/org/techpleiad/plato/api/response/ServicesAcrossBranchValidateResponseTO.java:15]: Try to eliminate  @such not required commits @DialgicMew

File: plato-api/src/main/java/org/techpleiad/plato/api/response/BranchProfileReportResponseTO.java [hunk 2]
```diff
@@ -1,18 +1,19 @@
...
 
 import java.util.List;
 
+@JsonInclude(JsonInclude.Include.NON_NULL)
 @Builder
 @Getter
 @ToString
 public class BranchProfileReportResponseTO {
     private String profile;
     private boolean fileEqual;
```
Comment (ashishgopalhattimare) [inline plato-api/src/main/java/org/techpleiad/plato/api/response/BranchProfileReportResponseTO.java:16]: here is it primitive data type @DialgicMew

File: plato-core/src/main/java/org/techpleiad/plato/core/domain/BranchProfileReport.java [hunk 3]
```diff
@@ -11,8 +11,7 @@
 @Getter
 public class BranchProfileReport {
     private String profile;
-    private boolean fileEqual;
-    private boolean propertyValueEqual;
-
+    private Boolean fileEqual;
```
Comment (ashishgopalhattimare) [inline plato-core/src/main/java/org/techpleiad/plato/core/domain/BranchProfileReport.java:14]: Here is it Object type @DialgicMew

File: plato-api/src/main/java/org/techpleiad/plato/api/response/ServicesAcrossBranchValidateResponseTO.java (hunk 1 above)
Comment (DialgicMew) [reply plato-api/src/main/java/org/techpleiad/plato/api/response/ServicesAcrossBranchValidateResponseTO.java:15]: yes

File: plato-core/src/main/java/org/techpleiad/plato/core/domain/BranchProfileReport.java (hunk 3 above)
Comment (DialgicMew) [reply plato-core/src/main/java/org/techpleiad/plato/core/domain/BranchProfileReport.java:14]: fixed to primitive

**Pull request merged**